README.md

```diff
@@ -90,6 +90,9 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [Gitignore.io](https://www.gitignore.io) Create Useful .gitignore Files For Your Projects
 * [Exercism](http://exercism.io/) Code practice and mentorship for everyone
 
+### Android developpement
+* [Android Development Track - One Million Arab Coders](https://classroom.udacity.com/nanodegrees/nd803-1mac-v2/dashboard/overview) : Nanodegree in Android developpement ! 
```

Comment:
This link isn't working for us, the non-registred to 1M Arab coders, please add another resource (better than a link to a course)